tests/unit/cluster/slot-stats.tcl

```diff
@@ -271,10 +530,17 @@ start_cluster 1 0 {tags {external:skip cluster}} {
 # -----------------------------------------------------------------------------
 
 start_cluster 1 1 {tags {external:skip cluster}} {
```

Comment:
```suggestion
start_cluster 1 1 {tags {external:skip cluster} overrides {cluster-slot-stats-enabled yes}{ {
```
Would be the more normal way to start up the cluster with the config value already set.